roles/edpm_ovn/defaults/main.yml

```diff
@@ -61,8 +61,12 @@ edpm_ovn_controller_common_volumes:
   - /var/lib/kolla/config_files/ovn_controller.json:/var/lib/kolla/config_files/config.json:ro
 
 edpm_ovn_controller_tls_volumes:
-  - /etc/pki/tls/certs/:/etc/pki/tls/certs/
-  - /etc/pki/tls/private/:/etc/pki/tls/private/
+  - "/var/lib/openstack/certs/{{ edpm_service_name|default('ovn') }}/ca.crt:/etc/pki/tls/certs/ovndbca.crt:ro,z"
+  - "/var/lib/openstack/certs/{{ edpm_service_name|default('ovn') }}/tls.crt:/etc/pki/tls/certs/ovndb.crt:ro,z"
+  - "/var/lib/openstack/certs/{{ edpm_service_name|default('ovn') }}/tls.key:/etc/pki/tls/private/ovndb.key:ro,Z"
+  - "/var/lib/openstack/cacerts/{{ edpm_service_name|default('ovn') }}/tls-ca-bundle.pem:/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem:ro,z"
```

Comment:
i assume this is requried for the webhook callback form ovn to the neuron server to enable it to send events to neutron.

Reply:
it's not required for anything specific, the cabundle just should be trusted everywhere